interface/globals.php

```diff
@@ -209,11 +209,22 @@ function undoMagicQuotes($array, $topLevel=true) {
         }
       }
     }
-    if ($gl_name == 'language_menu_other') {
+    if ($gl_name == 'language_menu_other') {       
       $GLOBALS['language_menu_show'][] = $gl_value;
     }
     else if ($gl_name == 'css_header') {
       $GLOBALS[$gl_name] = "$rootdir/themes/" . $gl_value;
+      
+      // For RTL languages, substitute the same theme converted for RTL// 
+      if ( isset( $_SESSION['language_direction'] ) &&
+              $_SESSION['language_direction'] == 'rtl' && 
+              !strpos($GLOBALS['css_header'], 'rtl')  ) {
+        $GLOBALS['css_header'] = "$rootdir/themes/rtl_" . $gl_value; // str_replace( 'themes/style_', 'themes/rtl_style_', $GLOBALS['css_header'] );
```

Comment:
To safely do above, you may need to move this entire block of code below in the Language Cleanup Section. To simplify, could store the $gl_value in a variable to use then use below.